src/components/RelatedTitleInfo/RelatedTitleInfo.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import {
+  Col,
+  Headline,
+  KeyValue,
+  NoValue,
+  Row,
+} from '@folio/stripes/components';
+
+import { resourceTypes } from '../../constants';
+import EResourceIdentifier from '../EResourceIdentifier';
+import css from './RelatedTitle.css';
+
+const propTypes = {
+  relatedTitles: PropTypes.arrayOf(PropTypes.shape({
+    titleInstance: PropTypes.shape({
+      id: PropTypes.string,
+      identifiers: PropTypes.arrayOf(PropTypes.shape({
+        identifier: PropTypes.shape({
+          ns: PropTypes.shape({
+            value: PropTypes.string,
+          }),
+          value: PropTypes.string,
+        }),
+      })),
+      longName: PropTypes.string,
+      name: PropTypes.string,
+      subType: PropTypes.shape({
+        id: PropTypes.string,
+        label: PropTypes.string,
+        value: PropTypes.string,
+      }),
+      type: PropTypes.shape({
+        id: PropTypes.string,
+        label: PropTypes.string,
+        value: PropTypes.string,
+      }),
+    }),
+  })).isRequired,
+};
+
+const RelatedTitleInfo = ({
+  relatedTitles
+}) => (
+  relatedTitles.map((titleInstance, i) => (
+    <div key={`related-title-${i}`} data-test-related-titles>
+      <div className={css.separator} />
```

Comment:
Sorry I don't understand what you mean with that.

Reply:
> We shouldn't be rendering a separator for the last item in a list, this renders the separator unconditionally.
> 
Hm, but in this case we have to render the separator at the beginning of a related title section, we have to start with the separator because the title info always comes first and has no separator at its end
> Maybe adding that functionality _will_ give you a reason to keep this component as something that renders an entire array rather than my suggestion above.
yes I think so